service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -84,7 +84,7 @@ private ControlledResourceKeys() {}
 
     // AWS
     public static final String AWS_CLOUD_CONTEXT = "awsCloudContext";
-    public static final String AWS_ENVIRONMENT_USER_ROLE_ARN = "awsEnvironmentUserRoleArn";
+    public static final String AWS_ENVIRONMENT_NOTEBOOK_ROLE_ARN = "awsEnvironmentUserRoleArn";
```

Comment:
```suggestion
    public static final String AWS_ENVIRONMENT_USER_ROLE_ARN = "awsEnvironmentUserRoleArn";
    public static final String AWS_ENVIRONMENT_NOTEBOOK_ROLE_ARN = "awsEnvironmentNotebookRoleArn";
```

Reply:
I don't really know the chances of hitting some startup time deserialization error that ends up triggering a manual cleanup.  Will that only happen if a notebook is in the process of being created when we're deploying?  We know notebook creations are long running, and I'd like to de-risk this.

@ddietterich any thoughts on risk?

Reply:
These keys are only used during the flight from what i see. whatever notebooks we have now in devel - i can clean those workspaces and/or notebooks if needed.

Reply:
Not sure how relevant this is, but ...

```sql
SELECT status, COUNT(status) 
FROM flight 
WHERE flightid IN (
    SELECT flightid 
    FROM flightinput 
    WHERE key = 'awsEnvironmentUserRoleArn'
) 
GROUP BY status;
```
Result:
```
 status  | count 
---------+-------
 ERROR   |    11
 FATAL   |     3
 SUCCESS |    45
(3 rows)
```

Reply:
At this time, I am not being careful about changing flight map keys and parameters. My judgment is that the complexity of handling the case is high and the risk is very low right now. The risk is catching a running flight during a rolling upgrade and hitting a serdes failure. I have yet to see an instance of that failure.

I am trying to be careful about changing serialized metadata in the WSM database. (Although given last week's debacle, not quite careful enough).

Reply:
Thanks Dan.

Dan and I talked offline; only flights using this key with `status` of `RUNNING` would trigger deserialization issues; the rest in the query above are kept for a period for debugging/forensics and would only be browsed in the database directly.  So it sounds like it's safe enough to do what @dexamundsen is proposing and babysit the deployment to `devel`.  I'll accept the suggested change.